src/CMakeLists.txt

```diff
@@ -23,6 +23,17 @@ CONFIGURE_FILE(
     @ONLY
     )
 
+EXECUTE_PROCESS(COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/hdf5_back_gen.py "VAL_TO_BUF_H" OUTPUT_VARIABLE HDF5_BACK_CC_VAL_TO_BUF_H RESULT_VARIABLE res_var_h)
+IF(NOT "${res_var_h}" STREQUAL "0")
+  message(FATAL_ERROR "Process hdf5_back_gen.py 'VAL_TO_BUF_H' failed, result = '${res_var_h}'")
+ENDIF()
+
+CONFIGURE_FILE(
+    "${CMAKE_CURRENT_SOURCE_DIR}/hdf5_back.h.in"
+    "${CMAKE_CURRENT_SOURCE_DIR}/hdf5_back.h"
```

Comment:
After merging from master, can you please update this to match the *.gen* pattern the hdf5back.cc uses.